### ISSUE TITLE
Add custom replacements to sanitizeFileName

### ISSUE THREAD

**Issue by christianromeni** [contao/core#7904]
It would be nice to configure the characters to be replaced in file and folder sanitize.
In this way, we could prevent file names like "this file has spaces.jpg" or "some_sonderzeichän.jpg"
The same thing should be possible with folders.

Alternatively it could be possible just to use something like standardize(). Since it actually does exactly what should be done.

**Comment by aschempp** [comment]
We've discussed this for quite some time. Standardize will change all files to latin/ascii characters, which is certainly not a world wide solution. People in China want their file names in chinese, not in [Pinyin](https://en.wikipedia.org/wiki/Pinyin)

**Comment by christianromeni** [comment]
Agreed on that... Thats why it should be optional... maybe just a checkbox in the config stating: "Standardise File and Foldernames on Upload/create"
